{
  "uid" : "e3d44ca5516971b5",
  "name" : "Test submenu and button on the Webinars and Events page",
  "fullName" : "Test Redirection of Pages: Test submenu and button on the Webinars and Events page",
  "historyId" : "47630eb91979b618063c05268ce85685",
  "time" : {
    "start" : 1698820203207,
    "stop" : 1698820240703,
    "duration" : 37496
  },
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"/html/body/div[1]/div[3]/div/div/div/div/main/article/div/section/div[6]/div/div/div/div[2]/a\"}\n  (Session info: chrome=114.0.5735.133); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55dfae4ab4e3 <unknown>\n#1 0x55dfae1dac76 <unknown>\n#2 0x55dfae216c96 <unknown>\n#3 0x55dfae216dc1 <unknown>\n#4 0x55dfae2507f4 <unknown>\n#5 0x55dfae23603d <unknown>\n#6 0x55dfae24e30e <unknown>\n#7 0x55dfae235de3 <unknown>\n#8 0x55dfae20b2dd <unknown>\n#9 0x55dfae20c34e <unknown>\n#10 0x55dfae46b3e4 <unknown>\n#11 0x55dfae46f3d7 <unknown>\n#12 0x55dfae479b20 <unknown>\n#13 0x55dfae470023 <unknown>\n#14 0x55dfae43e1aa <unknown>\n#15 0x55dfae4946b8 <unknown>\n#16 0x55dfae494847 <unknown>\n#17 0x55dfae4a4243 <unknown>\n#18 0x7f758e890402 <unknown>\n\n",
  "statusTrace" : "  File \"/usr/local/lib/python3.10/dist-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/usr/local/lib/python3.10/dist-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"steps/PageRedirectionSteps.py\", line 134, in step_impl\n    button_element = context.driver.find_element(By.XPATH, xpath)\n  File \"/home/addweb/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 740, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n  File \"/home/addweb/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 346, in execute\n    self.error_handler.check_response(response)\n  File \"/home/addweb/.local/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py\", line 245, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "testStage" : {
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"/html/body/div[1]/div[3]/div/div/div/div/main/article/div/section/div[6]/div/div/div/div[2]/a\"}\n  (Session info: chrome=114.0.5735.133); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55dfae4ab4e3 <unknown>\n#1 0x55dfae1dac76 <unknown>\n#2 0x55dfae216c96 <unknown>\n#3 0x55dfae216dc1 <unknown>\n#4 0x55dfae2507f4 <unknown>\n#5 0x55dfae23603d <unknown>\n#6 0x55dfae24e30e <unknown>\n#7 0x55dfae235de3 <unknown>\n#8 0x55dfae20b2dd <unknown>\n#9 0x55dfae20c34e <unknown>\n#10 0x55dfae46b3e4 <unknown>\n#11 0x55dfae46f3d7 <unknown>\n#12 0x55dfae479b20 <unknown>\n#13 0x55dfae470023 <unknown>\n#14 0x55dfae43e1aa <unknown>\n#15 0x55dfae4946b8 <unknown>\n#16 0x55dfae494847 <unknown>\n#17 0x55dfae4a4243 <unknown>\n#18 0x7f758e890402 <unknown>\n\n",
    "statusTrace" : "  File \"/usr/local/lib/python3.10/dist-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/usr/local/lib/python3.10/dist-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"steps/PageRedirectionSteps.py\", line 134, in step_impl\n    button_element = context.driver.find_element(By.XPATH, xpath)\n  File \"/home/addweb/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 740, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n  File \"/home/addweb/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 346, in execute\n    self.error_handler.check_response(response)\n  File \"/home/addweb/.local/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py\", line 245, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
    "steps" : [ {
      "name" : "Given The user is on the \"stg\" site",
      "time" : {
        "start" : 1698820204155,
        "stop" : 1698820215281,
        "duration" : 11126
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "When The user hovers over the \"resources\" menu",
      "time" : {
        "start" : 1698820215281,
        "stop" : 1698820215628,
        "duration" : 347
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "And The user clicks on the \"webinar\" submenu link and verify the redirection \"webinar\" page",
      "time" : {
        "start" : 1698820215628,
        "stop" : 1698820227026,
        "duration" : 11398
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "When The user clicks on the \"register_now\" button verify the page redirection",
      "time" : {
        "start" : 1698820227027,
        "stop" : 1698820235439,
        "duration" : 8412
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "When The user clicks on the \"view_now_1\" button verify the page redirection",
      "time" : {
        "start" : 1698820235440,
        "stop" : 1698820240630,
        "duration" : 5190
      },
      "status" : "broken",
      "statusMessage" : "selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"/html/body/div[1]/div[3]/div/div/div/div/main/article/div/section/div[6]/div/div/div/div[2]/a\"}\n  (Session info: chrome=114.0.5735.133); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55dfae4ab4e3 <unknown>\n#1 0x55dfae1dac76 <unknown>\n#2 0x55dfae216c96 <unknown>\n#3 0x55dfae216dc1 <unknown>\n#4 0x55dfae2507f4 <unknown>\n#5 0x55dfae23603d <unknown>\n#6 0x55dfae24e30e <unknown>\n#7 0x55dfae235de3 <unknown>\n#8 0x55dfae20b2dd <unknown>\n#9 0x55dfae20c34e <unknown>\n#10 0x55dfae46b3e4 <unknown>\n#11 0x55dfae46f3d7 <unknown>\n#12 0x55dfae479b20 <unknown>\n#13 0x55dfae470023 <unknown>\n#14 0x55dfae43e1aa <unknown>\n#15 0x55dfae4946b8 <unknown>\n#16 0x55dfae494847 <unknown>\n#17 0x55dfae4a4243 <unknown>\n#18 0x7f758e890402 <unknown>\n\n",
      "statusTrace" : "  File \"/usr/local/lib/python3.10/dist-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/usr/local/lib/python3.10/dist-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"steps/PageRedirectionSteps.py\", line 134, in step_impl\n    button_element = context.driver.find_element(By.XPATH, xpath)\n  File \"/home/addweb/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 740, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n  File \"/home/addweb/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 346, in execute\n    self.error_handler.check_response(response)\n  File \"/home/addweb/.local/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py\", line 245, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
      "steps" : [ ],
      "attachments" : [ {
        "uid" : "20bbe8c4dc5323d8",
        "name" : "Error Message",
        "source" : "20bbe8c4dc5323d8.txt",
        "type" : "text/plain",
        "size" : 936
      }, {
        "uid" : "ae1966e9dce52316",
        "name" : "Screenshot of https://stg.voltactivedata.com/events/ Page",
        "source" : "ae1966e9dce52316.png",
        "type" : "image/png",
        "size" : 112510
      } ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 2,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    }, {
      "name" : "When The user clicks on the \"view_now_2\" button verify the page redirection",
      "time" : {
        "start" : 1698820240703,
        "stop" : 1698820240703,
        "duration" : 0
      },
      "status" : "skipped",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "When The user clicks on the \"view_now_3\" button verify the page redirection",
      "time" : {
        "start" : 1698820240703,
        "stop" : 1698820240703,
        "duration" : 0
      },
      "status" : "skipped",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 7,
    "attachmentsCount" : 2,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "severity",
    "value" : "normal"
  }, {
    "name" : "feature",
    "value" : "Test Redirection of Pages"
  }, {
    "name" : "framework",
    "value" : "behave"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ ]
  },
  "source" : "e3d44ca5516971b5.json",
  "parameterValues" : [ ]
}